publication-commons/src/main/java/no/unit/nva/publication/service/impl/ResourceService.java

```diff
@@ -321,6 +321,27 @@ public Entity migrate(Entity dataEntry) {
         return dataEntry;
     }
 
+    public List<FileEntry> fetchFileEntriesForResource(Resource resource) {
```

Comment:
we need this function in addition to getFileEntries() because the dynamodbevent does not include file information and it is not possible to look up the resource the the service after its deleted

Reply:
I didn't look, but are we consuming the list and doing List::toStream, or is the list being used as a list?